src/components/AppRoutes/AppRoutes.jsx

```diff
@@ -0,0 +1,51 @@
+import { createBrowserRouter, Outlet } from 'react-router-dom'
+import Header from '../Header/Header'
+import Footer from '../Footer/Footer'
+import HomePage from '../../pages/HomePage/HomePage'
+import NotFoundPage from '../NotFoundPage/NotFoundPage'
+
+const routes = createBrowserRouter([

```

Comment:
```suggestion
const router = createBrowserRouter([
```
shall we name it router?

Reply:
and the component file itself : `src/components/AppRouter/AppRouter.jsx`